tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -8,27 +8,75 @@ Feature: full text search
+    And user "Alice" has created folder "personalFolderWithFile"
+    And user "Alice" has created folder "personalFolderWithFile/subFolder"
+    And user "Alice" has uploaded file with content "tagged file" to "personalFile.txt"
+    And user "Alice" has uploaded file with content "non-tagged file" to "fileWithoutTag.txt"
+    And user "Alice" has uploaded file with content "tagged file in folder" to "personalFolderWithFile/personalFileInsideFolder.txt"
+    And user "Alice" has uploaded file with content "tagged file in subfolder" to "personalFolderWithFile/subFolder/personalFileInsideSubFolder.txt"
+    And user "Alice" has created the following tags for file "personalFile.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for file "personalFolderWithFile/personalFileInsideFolder.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for file "personalFolderWithFile/subFolder/personalFileInsideSubFolder.txt" of the space "Personal":
+      | tag1 |
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "tag-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "spacesFolderWithFile/spacesSubFolder" in space "tag-space"
+    And user "Alice" has uploaded a file inside space "tag-space" with content "tagged file" to "spacesFile.txt"
+    And user "Alice" has uploaded a file inside space "tag-space" with content "non-tagged file" to "spacesFileWithoutTag.txt"
+    And user "Alice" has uploaded a file inside space "tag-space" with content "tagged file in folder" to "spacesFolderWithFile/spacesFileInsideFolder.txt"
+    And user "Alice" has uploaded a file inside space "tag-space" with content "tagged file in subfolder" to "spacesFolderWithFile/spacesSubFolder/spacesFileInsideSubFolder.txt"
+    And user "Alice" has created the following tags for file "spacesFile.txt" of the space "tag-space":
+      | tag1 |
+    And user "Alice" has created the following tags for file "spacesFolderWithFile/spacesFileInsideFolder.txt" of the space "tag-space":
+      | tag1 |
+    And user "Alice" has created the following tags for file "spacesFolderWithFile/spacesSubFolder/spacesFileInsideSubFolder.txt" of the space "tag-space":
+      | tag1 |
+    And using <dav-path-version> DAV path
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | personalFile.txt                |
+      | personalFileInsideFolder.txt    |
+      | personalFileInsideSubFolder.txt |
+      | spacesFile.txt                  |
+      | spacesFileInsideFolder.txt      |
+      | spacesFileInsideSubFolder.txt   |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
```

Comment:
what about spaces web dav? it should work right?

Reply:
While searching files in project space using spaces davpath the result came zero result as like in searching the share files using spaces dav path. Issues like this https://github.com/owncloud/ocis/issues/6669

Reply:
is there a issue for the project spaces too? if your sure it doesn't work please create a seperate issue for project sapces too or update the mentioned issue to include that the behaviour is same for project space as well.